examples/linux/io_epoll_test.cpp

```diff
@@ -87,6 +117,115 @@ int main() {
   } catch (const std::exception& ex) {
     std::printf("error: %s\n", ex.what());
   }
+
+  auto pipe_bench = [](auto& rPipeRef, auto& buffer, auto scheduler, int seconds, auto& data, auto& reps, auto& offset) {
+    return defer(
```

Comment:
I take it this `defer()` is so that the evaluation of `now(scheduler)` below is deferred until the operation actually starts?

Reply:
yes